In [3]:
import pandas as pd
import numpy as np

In [4]:
! ls

Data Exploration.ipynb	sample_submission.csv  test.zip  train.csv
resources		test		       train	 train.zip


In [7]:
df = pd.read_csv('train.csv')

In [8]:
df.head()

Image         Id
0  0000e88ab.jpg  w_f48451c
1  0001f9222.jpg  w_c3d896a
2  00029d126.jpg  w_20df2c5
3  00050a15a.jpg  new_whale
4  0005c1ef8.jpg  new_whale

In [13]:
only_one = df.Id.value_counts() == 1

In [17]:
only_one.value_counts()

False    2932
True     2073
Name: Id, dtype: int64

In [18]:
len(only_one)

5005

number of whales: 5005
whales with only one image: 2073